### ISSUE TITLE
Build failes for https://f-droid.org/repo/org.y20k.transistor_77.apk with 2.4.2-d3f9d5-SNAPSHOT + 2.4.1

### ISSUE THREAD

**Issue by MarkusML** [iBotPeaches/Apktool#2391]
### Information

1. 2.4.2-d3f9d5-SNAPSHOT + 2.4.1
2. Linux openSUSE 15.1
3. https://f-droid.org/repo/org.y20k.transistor_77.apk

### Steps to Reproduce

wget https://f-droid.org/repo/org.y20k.transistor_77.apk

rm -f /home/titan/.local/share/apktool/framework/1.apk # rule out this potential cause

apktool -version # 2.4.2-d3f9d5-SNAPSHOT

apktool d org.y20k.transistor_77.apk

apktool b -o org.y20k.transistor_77_new.apk org.y20k.transistor_77

I: Using Apktool 2.4.2-d3f9d5-SNAPSHOT
I: Checking whether sources has changed...
I: Smaling smali folder into classes.dex...
I: Checking whether resources has changed...
I: Building resources...
brut.androlib.AndrolibException: brut.common.BrutException: could not exec (exit code = 1): [/tmp/brut_util_Jar_86515815810963850775862730944781997552.tmp, p, --forced-package-id, 127, --min-sdk-version, 25, --target-sdk-version, 29, --version-code, 77, --version-name, 4.0.5, --no-version-vectors, -F, /tmp/APKTOOL14215150854354470238.tmp, -e, /tmp/APKTOOL6637640931856801549.tmp, -0, arsc, -I, /home/titan/.local/share/apktool/framework/1.apk, -S, /home/titan/tmp3/org.y20k.transistor_77/res, -M, /home/titan/tmp3/org.y20k.transistor_77/AndroidManifest.xml]
W: invalid resource directory name: /home/titan/tmp3/org.y20k.transistor_77/res navigation

=> Result: Exit code 1 and no new apk file is created.

### Frameworks
none

### APK
see above

### Questions to ask before submission
1. Have you tried `apktool d`, `apktool b` without changing anything? => yes, see above
2. If you are trying to install a modified apk, did you resign it? => it's a build problem
3. Are you using the latest apktool version? => very latest version (2.4.2-d3f9d5-SNAPSHOT) as well as older version (2.4.1) show same problem

**Comment by AndnixSH** [comment]
I have the same excat error `W: invalid resource directory name:`  for this APK https://apkcombo.com/en-dk/apk-downloader/?device=&arch=&android=&q=com.herogame.gplay.evolutionchess

```
PS D:\> java -jar apktool.jar d D:\arena-of-evolution-red-tides_1.9.14.apk
I: Using Apktool 2.4.1 on arena-of-evolution-red-tides_1.9.14.apk
I: Loading resource table...
I: Decoding AndroidManifest.xml with resources...
I: Loading resource table from file: C:\Users\bent\AppData\Local\apktool\framework\1.apk
I: Regular manifest package...
I: Decoding file-resources...
I: Decoding values */* XMLs...
I: Baksmaling classes.dex...
I: Baksmaling classes2.dex...
I: Baksmaling classes3.dex...
I: Copying assets and libs...
I: Copying unknown files...
I: Copying original files...

PS D:\> java -jar apktool.jar b D:\arena-of-evolution-red-tides_1.9.14
I: Using Apktool 2.4.1
I: Checking whether sources has changed...
I: Smaling smali folder into classes.dex...
I: Checking whether sources has changed...
I: Smaling smali_classes2 folder into classes2.dex...
I: Checking whether sources has changed...
I: Smaling smali_classes3 folder into classes3.dex...
I: Checking whether resources has changed...
I: Building resources...
W: invalid resource directory name: D:\arena-of-evolution-red-tides_1.9.14\res navigation
brut.androlib.AndrolibException: brut.common.BrutException: could not exec (exit code = 1): [C:\Users\bent\AppData\Local\Temp\brut_util_Jar_7273438890981463980.tmp, p, --forced-package-id, 127, --min-sdk-version, 17, --target-sdk-version, 28, --version-code, 10914, --version-name, 1.9.14, --no-version-vectors, -F, C:\Users\bent\AppData\Local\Temp\APKTOOL2968749797619490565.tmp, -e, C:\Users\bent\AppData\Local\Temp\APKTOOL9837391649008694804.tmp, -0, arsc, -I, C:\Users\bent\AppData\Local\apktool\framework\1.apk, -S, D:\arena-of-evolution-red-tides_1.9.14\res, -M, D:\arena-of-evolution-red-tides_1.9.14\AndroidManifest.xml]
PS D:\>
```

**Comment by giacomoferretti** [comment]
Duplicate of #1811 #2219 #2338

**TL;DR** use `--use-aapt2` to fix this problem. You will get some warnings but at least it rebuilds the apk.
